paths/release_triggers/create.yaml

```diff
@@ -0,0 +1,91 @@
+---
+summary: Create a release trigger
+description: Create a new recurring release. New releases will be published automatically, based on the cron schedule provided.
+  Currently, only one release trigger can exist per distribution.
+operationId: release_triggers/create
+tags:
+  - Release Triggers
+parameters:
+  - "$ref": "../../parameters.yaml#/X-PhraseApp-OTP"
+  - "$ref": "../../parameters.yaml#/account_id"
+  - "$ref": "../../parameters.yaml#/distribution_id"
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
+        type: object
+        title: release/create/parameters
+        properties:
+          cron_schedule:
+            description: Cron schedule for the scheduler. Read more about the format of this field at https://en.wikipedia.org/wiki/Cron
+            type: string
+            example: "15 18 1,3 * *"
```

Comment:
Not something that should block this PR, but since we don't offer a variety of scheduling options specifically around days of months/months, should our examples also reflect it in our example? In our case, that would mean the third and fourth entry of cron schedule will always be "*"

e.g. 00 11 * * 0,4,5,6

Reply:
Well on a second thought, I guess this shouldn't be an issue, allowing user a flexibility via the API, this should be translated into appropriate next_run_at by Fugit without issues and release will be triggered accordingly ideally.. So I guess if anything this is more of a flexibility that is on API that is not present on UI. What do you think?

Reply:
hm IMO we should offer the same features through UI and API, so e.g. not allow triggering stuff every minute etc.

not sure how easy this would be to validate (and reject the request if needed), perhaps fugit offers something...

Reply:
Like having a validation to raise error when cron has for example entries for month/days of month other than *?

Reply:
yeah, or multiple minutes/hours given etc

Reply:
I adapted the example already, I think

Reply:
yes you did! my bad missed it.